public/assets/css/navbar.css

```diff
@@ -0,0 +1,61 @@
+/* header */
+*{
+    margin: 0;
+}
+.banner{
+    width: 100%;
+    height: 180px;
+    display: flex;
+    align-items: end;
+    justify-content: center;
+    background-image: url("../images/banner.jpg");
+    background-position: center;
+    background-size: cover;
+    text-shadow: 2px 2px 2px var(--color-main);
+}
+.banner h1 {
+    color: var(--color-light);
+    font-family: var(--font-banner);
+    font-size: var(--sizeBanner);
+    flex-wrap: wrap;
+    justify-content: space-around;
+}
+.bg-dark {
+    background-color:var(--color-main)!important;
```

Comment:
pas de !important dans votre CSS, c'est une mauvaise pratique

Reply:
à modifier partout ;-)
si tu veux surcharger Bootstrap, il faut que tu utilise un selecteur de meme spécifité ;-)

Reply:
All is removed !